content/blog/2023/10/kubewarden-1.8-release.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubewarden 1.8.0 release
+authors:
+  - Víctor Cuadrado
+date: 2023-10-11
+---
+
+Today we are happy to announce the release of Kubewarden 1.8.0! 🎉🥳
+
+This is a small release, focused on [OpenTelemetry](https://opentelemetry.io/).
+
+The OpenTelemetry Protocol (OTLP) got its [first 1.0.0
+version](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.0.0)
+in July 2023; several
+libraries got their first 1.0.0 release, such as the [Go metric
+SDK](https://opentelemetry.io/blog/2023/otel-go-metrics-sdk-stable/) or the
+[.NET Automatic
+Instrumentation](https://opentelemetry.io/blog/2023/otel-dotnet-auto-instrumentation/).
+
+Still, the OpenTelemetry stack is not yet stable, and unannounced backwards-incompatible
+changes still happen. You can have a look at the status of each of their
+libraries and protocols [here](https://opentelemetry.io/status).
+
+For Kubewarden, this means we should expect more users to configure and fine
+tune the deployment to fit their stacks.
+
+## Changes
+
+With 1.8.0, the Kubewarden Controller now separately reconciles the configuration of both
+tracing and metrics for Policy Server instances. In the past, enabling metrics in
+Kubewarden meant enabling both, and now users can decide to enable each separately, as we
+have realized from the requests raised by the community.
+
+We have made the concious decision to make this a breaking change in the helm
+chart `values.yaml`, which is uncommon.
+We think it simplifies the adoption of OTLP changes in the future.
+
+In addition, since `v0.85.0` OpenTelemetry Collector no longer includes
+exporters for the native Jaeger format. This change was
+prompted because Jaeger has support for OTLP. See the [announcement
+here](https://opentelemetry.io/blog/2023/jaeger-exporter-collector-migration).
+
+Hence we now take care of setting `exporters.otlp/jaeger` instead of
+`exporters.jaeger` the OpenTelemetryCollector CR we create. For the user, this
```

Comment:
```suggestion
Now the OpenTelemetryCollector resource created by the helm chart is configured
to use  `exporters.otlp/jaeger` instead of `exporters.jaeger`. For the user, this
```